<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SponsorshipModule</class>
 <widget class="QMainWindow" name="SponsorshipModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SR Studio - Partner &amp; Sponsorship Network</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1160</width>
      <height>780</height>
     </rect>
    </property>
    
    <!-- Partner Hub -->
    <widget class="QWidget" name="tab_hub">
      <attribute name="title"><string>ü§ù PARTNER HUB</string></attribute>
      <!-- List Area (Left) -->
      <widget class="QGroupBox" name="groupBox_list">
       <property name="geometry"><rect><x>10</x><y>10</y><width>750</width><height>730</height></rect></property>
       <property name="title"><string>ACTIVE PARTNERSHIPS</string></property>
       <widget class="QTableView" name="tabspons">
        <property name="geometry"><rect><x>10</x><y>100</y><width>730</width><height>620</height></rect></property>
       </widget>
       <widget class="QLineEdit" name="le_rech">
        <property name="geometry"><rect><x>10</x><y>40</y><width>250</width><height>40</height></rect></property>
        <property name="placeholderText"><string>Search partners...</string></property>
       </widget>
       <widget class="QPushButton" name="bp_tri">
        <property name="geometry"><rect><x>270</x><y>40</y><width>100</width><height>40</height></rect></property>
        <property name="text"><string>SORT</string></property>
       </widget>
       <widget class="QPushButton" name="bp_pdf">
        <property name="geometry"><rect><x>380</x><y>40</y><width>100</width><height>40</height></rect></property>
        <property name="text"><string>PDF</string></property>
       </widget>
      </widget>
      
      <!-- Details Form (Right) -->
      <widget class="QGroupBox" name="groupBox_details">
       <property name="geometry"><rect><x>770</x><y>10</y><width>370</width><height>730</height></rect></property>
       <property name="title"><string>CONTRACT CONFIGURATION</string></property>
       
       <widget class="QLabel" name="label_id"><property name="geometry"><rect><x>20</x><y>50</y><width>330</width><height>20</height></rect></property><property name="text"><string>PARTNER ID (AUTO)</string></property></widget>
       <widget class="QLineEdit" name="le_id"><property name="geometry"><rect><x>20</x><y>75</y><width>330</width><height>40</height></rect></property><property name="readOnly"><bool>true</bool></property></widget>
       
       <widget class="QLabel" name="label_n"><property name="geometry"><rect><x>20</x><y>125</y><width>330</width><height>20</height></rect></property><property name="text"><string>SPONSOR / BRAND NAME</string></property></widget>
       <widget class="QLineEdit" name="le_nom"><property name="geometry"><rect><x>20</x><y>150</y><width>330</width><height>40</height></rect></property></widget>
       
       <widget class="QLabel" name="label_d"><property name="geometry"><rect><x>20</x><y>200</y><width>160</width><height>20</height></rect></property><property name="text"><string>DURATION (S)</string></property></widget>
       <widget class="QLineEdit" name="le_duree"><property name="geometry"><rect><x>20</x><y>225</y><width>160</width><height>40</height></rect></property></widget>
       
       <widget class="QLabel" name="label_a"><property name="geometry"><rect><x>190</x><y>200</y><width>160</width><height>20</height></rect></property><property name="text"><string>AIRTIME (H)</string></property></widget>
       <widget class="QLineEdit" name="le_heure"><property name="geometry"><rect><x>190</x><y>225</y><width>160</width><height>40</height></rect></property></widget>
       
       <widget class="QLabel" name="label_r"><property name="geometry"><rect><x>20</x><y>275</y><width>330</width><height>20</height></rect></property><property name="text"><string>AD REPETITIONS</string></property></widget>
       <widget class="QSpinBox" name="le_repet_spin"><property name="geometry"><rect><x>20</x><y>300</y><width>330</width><height>40</height></rect></property></widget>
       
       <widget class="QPushButton" name="bp_ajouter">
        <property name="geometry"><rect><x>20</x><y>380</y><width>330</width><height>50</height></rect></property>
        <property name="text"><string>‚ûï REGISTER NEW PARTNER</string></property>
       </widget>
       <widget class="QPushButton" name="bp_modifier">
        <property name="geometry"><rect><x>20</x><y>440</y><width>330</width><height>50</height></rect></property>
        <property name="text"><string>üíæ SAVE CONTRACT CHANGES</string></property>
       </widget>
       <widget class="QPushButton" name="bp_supprime">
        <property name="geometry"><rect><x>20</x><y>500</y><width>330</width><height>50</height></rect></property>
        <property name="text"><string>üóëÔ∏è TERMINATE CONTRACT</string></property>
       </widget>
       <widget class="QPushButton" name="bp_clear">
        <property name="geometry"><rect><x>20</x><y>560</y><width>330</width><height>40</height></rect></property>
        <property name="text"><string>CLEAR WORKSPACE</string></property>
       </widget>

       <widget class="QLabel" name="mini_stat_label">
        <property name="geometry"><rect><x>20</x><y>610</y><width>330</width><height>110</height></rect></property>
        <property name="text"><string></string></property>
       </widget>
      </widget>
    </widget>

    <widget class="QWidget" name="tab_stats">
     <attribute name="title"><string>üìä REVENUE INSIGHTS</string></attribute>
     <widget class="QLabel" name="label_stat"><property name="geometry"><rect><x>20</x><y>20</y><width>1120</width><height>650</height></rect></property></widget>
     <widget class="QPushButton" name="bp_aller"><property name="geometry"><rect><x>480</x><y>680</y><width>200</width><height>45</height></rect></property><property name="text"><string>GENERATE ANALYTICS</string></property></widget>
    </widget>
    
    <widget class="QWidget" name="tab_clock">
     <attribute name="title"><string>üïí STUDIO TOOLS</string></attribute>
     
     <widget class="QGroupBox" name="groupBox_timer">
      <property name="geometry"><rect><x>300</x><y>150</y><width>560</width><height>350</height></rect></property>
      <property name="title"><string>AIRTIME COUNTDOWN</string></property>
      <widget class="QLCDNumber" name="mint"><property name="geometry"><rect><x>100</x><y>50</y><width>150</width><height>150</height></rect></property></widget>
      <widget class="QLCDNumber" name="secc"><property name="geometry"><rect><x>260</x><y>50</y><width>150</width><height>150</height></rect></property></widget>
      <widget class="QPushButton" name="bp_play"><property name="geometry"><rect><x>100</x><y>220</y><width>100</width><height>45</height></rect></property><property name="text"><string>START</string></property></widget>
      <widget class="QPushButton" name="bp_pause"><property name="geometry"><rect><x>210</x><y>220</y><width>100</width><height>45</height></rect></property><property name="text"><string>PAUSE</string></property></widget>
      <widget class="QPushButton" name="bp_stop"><property name="geometry"><rect><x>320</x><y>220</y><width>100</width><height>45</height></rect></property><property name="text"><string>RESET</string></property></widget>
     </widget>
    </widget>

   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar"/><widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
